chainercv/utils/testing/assertions/assert_is_instance_segmentation_dataset.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import six
+
+from chainercv.utils.testing.assertions.assert_is_image import assert_is_image
+
+
+def assert_is_instance_segmentation_dataset(dataset, n_class, n_example=None):
+    """Checks if a dataset satisfies instance segmentation dataset APIs.
+
+    This function checks if a given dataset satisfies instance segmentation
+    dataset APIs or not.
+    If the dataset does not satifiy the APIs, this function raises an
+    :class:`AssertionError`.
+
+    Args:
+        dataset: A dataset to be checked.
+        n_class (int): The number of classes including background.
+        n_example (int): The number of examples to be checked.
+            If this argument is specified, this function picks
+            examples ramdomly and checks them. Otherwise,
+            this function checks all examples.
+
+    """
+
+    assert len(dataset) > 0, 'The length of dataset must be greater than zero.'
+
+    if n_example:
+        for _ in six.moves.range(n_example):
+            i = np.random.randint(0, len(dataset))
+            _check_example(dataset[i], n_class)
+    else:
+        for i in six.moves.range(len(dataset)):
+            _check_example(dataset[i], n_class)
+
+
+def _check_example(example, n_class):
+    assert len(example) >= 4, \
+        'Each example must have at least four elements:' \
+        'img, bbox, mask and label.'
+
+    img, bbox, mask, label = example[:4]
+
+    assert_is_image(img, color=True)
+    _, H, W = img.shape
+    N = bbox.shape[0]
```

Comment:
Could you use `R` instead of `N` so that the notation is consistent with detection?